README.md

```diff
@@ -147,6 +152,15 @@ Use the [execution-role.yaml](examples/execution-role.yaml) file to generate an
 Alternatively, you can set the trust relationships of your role via AWS Console: in the IAM dashboard, select your role and click **Trust Relationships**:
 ![Screenshot 2023-03-31 at 17 32 55](https://user-images.githubusercontent.com/5663078/230436500-fb4ee057-b70e-4580-a94d-f56191728117.png)
```

Comment:
nit: i know it's not change in this PR, but better change the "screenshot..."

Reply:
Thanks for the comment, updated in 480ba7440fa549757773ca511ffa111ee5a2de56 ✅